# Inputs
string trajectory_name
---
# Outputs
bool accepted